src/styles/blocks/testimonials.scss

```diff
@@ -0,0 +1,240 @@
+.testimonials {
+  padding-bottom: 120px;
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__subhead {
+    margin-bottom: 16px;
+  }
+
+  &__title {
+    font-weight: 600;
+    font-size: 32px;
+    line-height: 150%;
+    color: #253757;
+    margin-bottom: 48px;
+  }
+
+  &__card {
+    width: 100%;
```

Comment:
Please use px. If, for example, two cards are removed, then the remaining one will stretch over the entire block.